go.mod

```diff
@@ -8,15 +8,22 @@ require (
 	github.com/docker/go-units v0.4.0
 	github.com/godbus/dbus/v5 v5.0.4
 	github.com/opencontainers/runtime-spec v1.0.2
+	github.com/shirou/gopsutil/v3 v3.22.12
```

Comment:
Is it hard to implement by ourselves? cgroups is Linux-only whereas gopsutil supports other OSes and bring a bunch of dependencies.